src/components/test/FollowersFollowing.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import FollowersFollowing from "../FollowersFollowing";
+import { shallow } from "enzyme";
+
+describe("Feed", () => {
+  it("renders without crashing", () => {
+    expect(() => {
+      shallow(<FollowersFollowing />);
+    }).not.toThrow();
+  });
+  it("to match snapshot", () => {
+    //write more specific tests when we are pulling from a real followers/following list
```

Comment:
After integrating with the redux graph (see above comment) those test can be written.